src/components/Hearing.js

```diff
@@ -167,21 +190,26 @@ class Hearing extends React.Component {
                 />
               </div>
             ))}
-            <div id="hearing-comments">
-              <CommentList
-                displayVisualization={userIsAdmin || hearing.closed}
-                section={mainSection}
-                comments={this.props.sectionComments[mainSection.id] ?
-                          this.props.sectionComments[mainSection.id].data : []}
-                canComment={mainSectionCommentable}
-                onPostComment={this.onPostHearingComment.bind(this)}
-                canVote={mainSectionVotable}
-                onPostVote={onPostVote}
-                canSetNickname={user === null}
-              />
-            </div>
-            <hr/>
-            <a href={reportUrl}><FormattedMessage id="downloadReport"/></a>
+            {fullscreenMapPlugin ?
+              null :
```

Comment:
Not sure how far this logic can stretch, but for now we just blatantly omit comment listing if Hearing has fullscreenMap. Not sure if we wan't to have fullscreen maps in the future and still have comments.